scripts/deployV1.1.0.ts

```diff
@@ -107,38 +108,22 @@ async function main() {
   ] as [string, string, string]
 
   console.log('Upgrading Feeds to v1.1.0')
-  const feedsFactory = await ethers.getContractFactory('Feeds')
-  const feedsContract = (await upgrades.upgradeProxy(
-    feedsProxyAddress as string,
-    feedsFactory
-  )) as Feeds
-  console.log('Feeds upgraded')
-  console.log(
-    await upgrades.erc1967.getImplementationAddress(feedsContract.address),
-    ' getImplementationAddress'
-  )
-  console.log(
-    await upgrades.erc1967.getAdminAddress(feedsContract.address),
-    ' getAdminAddress'
-  )
-  await feedsContract.initialize(...ketlGuardedConstructorArguments)
+  const feedsContract = await upgradeAndInitializeContract({
+    proxyAddress: feedsProxyAddress as string,
+    constructorArguments: ketlGuardedConstructorArguments,
+    contractName: 'Feeds',
+    chainName,
+    initializer: 'initialize', // KetlGuarded initializer
```

Comment:
Let's remove the comment

Reply:
Would prefer to keep this comment because it's unclear which initializer (and therefore, which constructor arguments need to match up).